gap/digraph.gi

```diff
@@ -1368,6 +1423,259 @@ function(filt, n, p)
   return DigraphNC(IsMutableDigraph, RANDOM_DIGRAPH(n, Int(p * 10000)));
 end);
 
+BindGlobal("DIGRAPHS_FillOutGraph", function(n, p, adjacencyList)
+
+    local i, j;
+
+    for i in [1 .. n] do
+        for j in [1 .. n] do
+            if (not (j in adjacencyList[i])) then
+                 if Float(Random([0 .. 99])/100) < Float(p) then
```

Comment:
Similarly, you might want to create `[0 .. 99]` somewhere, since this is created every time.

I don't think `Float` is needed here, is it?  GAP is happy to deal with rationals.

Reply:
Several more cases of `Float` below, which should also be checked.

Reply:
Looks like you do need Float after all!

But get rid of `Float(p)` everywhere, since `p` should always be a float.